table/pit-cs-tree.go

```diff
@@ -139,16 +139,27 @@ func (p *PitCsTree) RemoveInterest(pitEntry PitEntry) bool {
 // Originally implemented as FindOrInsertPIT()
 func (p *PitCsTree) FindInterestExactMatch(interest *ndn.Interest) PitEntry {
 	node := p.root.findExactMatchEntry(interest.Name())
-	for _, curEntry := range node.pitEntries {
-		if curEntry.CanBePrefix() == interest.CanBePrefix() && curEntry.MustBeFresh() == interest.MustBeFresh() {
-			return curEntry
+	if node != nil {
+		for _, curEntry := range node.pitEntries {
+			if curEntry.CanBePrefix() == interest.CanBePrefix() && curEntry.MustBeFresh() == interest.MustBeFresh() {
+				return curEntry
+			}
 		}
 	}
 	return nil
 }
 
 // Logic taken from FindPITFromData
-func (p *PitCsTree) FindInterestPrefixMatch(interest *ndn.Interest, token uint32) []PitEntry {
+// If we have interests /a and /a/b, a prefix search for /a/b will return
+// PitEntries for both /a and /a/b
```

Comment:
Does this comment accurately describe what `FindInterestPrefixMatch` is supposed to do? Want to make sure I understood it correctly.

Reply:
An Interest will only match CS entry with prefix match. Don't know why the function you described is needed.

Reply:
You mean that there's no point in this function? (Takes in an interest and returns all matching interests that are prefixes or exact matches of it).

Reply:
Yes. I think it is not being used.

Reply:
Function has been deleted.